daemon/daemon.go

```diff
@@ -559,10 +619,9 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 	d.tomb.Kill(d.serve.Shutdown(ctx))
 	cancel()
 
-	if !restartSystem {
+	if !restarting {
```

Comment:
this seems wrong now,  we should send stopping for RestartDaemon and RestartSocket

Reply:
yes you're right I missed this, fixed now